core/src/main/java/nl/nn/adapterframework/senders/IbisLocalSender.java

```diff
@@ -44,10 +44,17 @@
 import nl.nn.adapterframework.util.Misc;
 
 /**
- * Posts a message to another IBIS-adapter in the same IBIS instance.
- *
- * Returns exit.code as forward name to SenderPipe.
- *
+ * Posts a message to another IBIS-adapter in the same IBIS instance. If the callee exits with an &lt;<code>exit</code>&gt;
+ * that has state {@link nl.nn.adapterframework.core.PipeLine.ExitState#ERROR}, an error is considered to happen
+ * in the caller which means that the <code>exception</code> forward is followed if it is present.
+ * <p/>
+ * <p/>
+ * Returns exit.code as forward name to SenderPipe provided that exit.code can be parsed as integer.
+ * For example, if the called adapter has an exit state with code
+ * <code>2</code>, then the {@link nl.nn.adapterframework.pipes.SenderPipe} supports a forward with name <code>2</code>
+ * that is followed when the called adapter exits with the mentioned exit. This does not work if the code is for example <code>c2</code>.
```

Comment:
De extra regels die toevoegt zijn overbodig, PipeLineExit#SetExitCode is een integer. Je kan daar nooit een waarde anders dan een integer in stoppen. Zowel de frank-xsd als de configuration warnings zullen hier een melding over geven.

Reply:
Toen ik dit aan het uitproberen was leek een niet-integer code in eerste instantie te werken. Mogelijk had ik de FrankDoc.xsd er toen niet bij. Daarom denk ik dat het zin heeft om deze tekst te laten staan. Het zou leuk zijn als je niet-integer forwards kan introduceren in sub-adapters, maar dat kan niet. Ook een reden om dit expliciet te maken.

Reply:
Ik zie dat je wel een foutmelding krijgt in de console, maar dat de adapter goed opstart, dit komt overeen met hoe het werkt voor pipes en senders. Het is dus niet zo dat het werkt, maar dat het genegeerd wordt. 

Het valt mij ook op dat de foutmelding onhandig is dus die zal ik aanpassen. De xsd zou dit dan ook tegen moeten gaan houden.